panther_analysis_tool/log_schemas/user_defined.py

```diff
@@ -398,8 +398,11 @@ def _contains_schema_tests(filename: str) -> bool:
 
     # - "input" and "logtype" are expected to be always present
     # - at least one of "result", "results" fields is required
-    return {'input', 'logtype', 'result'}.issubset(fields) or \
-           {'input', 'logtype', 'results'}.issubset(fields)
+    return {"input", "logtype", "result"}.issubset(fields) or {
```

Comment:
Just a reminder that maybe we should add the formatter check in CI. I probably forgot to rerun it in this case.

Reply:
https://app.asana.com/0/1199906407795548/1200886091502394/f